mapusaurus/hmda/views.py

```diff
@@ -14,19 +15,23 @@ def loan_originations(request):
     lender_hierarchy = request.GET.get('lh')
     peers = request.GET.get('peers')
     geoids = get_censustract_geoids(request)
-    institution_selected = Institution.objects.filter(pk=institution_id).first()
+    institution_selected = Institution.objects.get(pk=institution_id)
```

Comment:
.get will throw a nasty error if institution_id is invalid. .filter().first() returns None. Surround with a try/except?